src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingStonemason.java

```diff
@@ -105,7 +105,10 @@ public boolean canRecipeBeAdded(final IToken token)
         if (storage.getPrimaryOutput().getItem() instanceof BlockItem)
         {
             final Item item = storage.getPrimaryOutput().getItem();
-            if (item.isIn(Tags.Items.STONE) || item.isIn(Tags.Items.COBBLESTONE) || item.isIn(ItemTags.STONE_BRICKS))
+            if (
```

Comment:
Preferbably please remove the new line here so that the `item.isIn....` is after the brackets